magento/pwa-studio
# [PWA-1544] Verify Products

## Description

To verify that the Page Builder Products content type renders & functions correctly on PWA Studio/Venia UI storefront in desktop and mobile for each content type property

## Related Issue







Closes [PWA-1544](https://jira.corp.magento.com/browse/PWA-1544): Verify Products.

## Acceptance





### Verification Stakeholders



### Specification




## Verification Steps

1. Cypress `products.spec.js` should pass:
- Go to `venia-integration-test` dir
- Run `yarn install`
- Run `yarn test`
- Run `products.spec.js` from cypress ui

2. Verify that the test fails:
- Change something in the mock data that would cause a visual difference and run the test again. 


#### Test scenario(s) for direct fix/feature





#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
